Makefile

```diff
@@ -0,0 +1,87 @@
+.PHONY: build-garage-ci build-garage-headless build-garage-nvidia \
+	run-garage-ci run-garage-headless run-garage-nvidia
+
+SHELL = bash
+# Path in host where the experiment data obtained in the container is stored
+DATA_PATH ?= $(shell pwd)/data
+# Set the environment variable MJKEY with the contents of the file specified by
+# MJKEY_PATH.
+MJKEY_PATH ?= ~/.mujoco/mjkey.txt
+# Date/time is appended to the container names
+DATE_TIME ?= $(shell date "+%Y_%m_%d_%H_%M_%S")
+# Prevent garage from exiting on config by making sure the personal config file
+# is already created
+CONFIG_PERSONAL := garage/config_personal.py
+
+build-garage-ci: TAG ?= rlworkgroup/garage-ci
+build-garage-ci: docker/docker-compose-headless.yml copy_config_personal
+	TAG=${TAG} \
+	docker-compose \
```

Comment:
does this properly use `--cache-from`? otherwise, builds in the CI will take a very long time.

Reply:
I have added the cache-from argument to the docker-compose files:
```